app/models/container_template.rb

```diff
@@ -12,4 +12,69 @@ class ContainerTemplate < ApplicationRecord
   serialize :objects, Array
 
   acts_as_miq_taggable
+
+  MIQ_ENTITY_MAPPING = {
```

Comment:
@simon3z I added this mapping (only for objects we collect & can instantiate)  so the automation side will be able to know in which table to look for the objects. 
We may consider adding a more general mapping to reuse it later on in different places. 
cc @moolitayer